config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -1796,6 +1796,12 @@
 # DEFAULT: none (null)
 # basiclti.provider.highly.trusted.consumers=
 
+ #Colon separated list of trusted email consumers -  will allow the Sakai as LTI provider to optionally 'trust' the consumer and lookup the userid
+ #based on the consumer provided user email address.
+ #e.g., basiclti.provider.email.trusted.consumers=lmsng.school.edu:another.school.edu.
```

Comment:
Can the same consumer be used for multiple different LTI tools?  If so then would all LTI tools for this consumer now be highly trusted?  It would not be good to change the trust level for other tools.